app/redux/node/reducer.js

```diff
@@ -19,13 +20,55 @@ const initialState = {
   nodeIpAddress: nodeConsts.DEFAULT_URL
 };
 
+let startUpDelay = 0;
```

Comment:
you can make it static in Network screen and pass to action as parameter
Anyway logic for ui display in single component doesn't look good in reducer. Better return it to component and use static fields for more bug protected code

Reply:
fix